src/javascript/app/pages/trade/analysis.js

```diff
@@ -89,14 +111,63 @@ const TradingAnalysis = (() => {
                 showExplanation();
             }
         }
+        if (current_tab) {
+            const el_to_show = getElementById(current_tab);
+            slideSelector(tab_selector_id, el_to_show);
+            const mobile_tab_header = document.getElementById('tab_mobile_header');
+            if (mobile_tab_header) mobile_tab_header.innerHTML = el_to_show.firstChild.innerHTML;
+        }
+        // workaround for underline during window resize
+        window.addEventListener('resize', () => {
```

Comment:
we should probably remove this event listener somewhere, maybe onUnload of trade page